pkg/openstackplaybookgenerator/job.go

```diff
@@ -44,6 +44,11 @@ func PlaybookJob(cr *ospdirectorv1beta1.OpenStackPlaybookGenerator, configHash s
 	volumeMounts := GetVolumeMounts(cr)
 	volumes := GetVolumes(cr)
 
+	cmd := []string{"/bin/bash", "/home/cloud-admin/create-playbooks.sh"}
+	if cr.Spec.Debug {
+		cmd = []string{"/bin/sleep", "360d"}
```

Comment:
should we use sleep infinity?

Reply:
Same question from me, but otherwise looks good.

Reply:
infinity it is